socorro/external/postgresql/base.py

```diff
@@ -85,7 +85,7 @@ def query(self, sql, params=None, error_message=None, connection=None):
             if not connection:
                 connection = self.database.connection()
                 fresh_connection = True
-            # self.context.logger.debug(connection.cursor.mogrify(sql, params))
+            # logger.debug(connection.cursor().mogrify(sql, params))
```

Comment:
This is funny and I have used it before too :) Why don't we allow it to be on for ```debug```, we can run production in ```error``` and below if we don't like it there? Seems like this is what log levels are intended for.